RELEASING.md

```diff
@@ -0,0 +1,76 @@
+# Release process
+
+The GAPIC generator is used by the .NET Google Cloud Libraries team
+in two ways:
+
+- From the [google-cloud-dotnet repo](https://github.com/googleapis/google-cloud-dotnet),
+  for both developer-local generation and OwlBot-postprocessor-regeneration
+- From the [googleapis repo](https://github.com/googleapis/google-cloud-dotnet)
```

Comment:
I think this should be https://github.com/googleapis/googleapis

Reply:
Whoops, yes - fixing.